fl4health/utils/load_data.py

```diff
@@ -78,8 +79,25 @@ def load_cifar10_data(
             transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)),
         ]
     )
+
     training_set: BaseDataset = Cifar10Dataset(data_dir, train=True, transform=transform)
-    validation_set: BaseDataset = Cifar10Dataset(data_dir, train=False, transform=transform)
+    validation_set: BaseDataset
+    if validation_portion == 0:
```

Comment:
Super minor, but maybe just do 0.0 here. Sometimes python/mypy is weird with into versus float representations.